README.md

```diff
@@ -31,6 +31,11 @@ Please be respectful of the people who have created these resources. If you want
     * [06-325 Numerical Methods and ML for ChE](https://github.com/ulissigroup/F22-06-325) - Carnegie Mellon University, Prof. Zachary Ulissi, Fall 2022.
     * [Numerical Methods and Statistics](https://github.com/whitead/numerical_stats) - University of Rochester, Prof. Andrew White.
     * [Python for Chemical Engineers](https://github.com/CAChemE/Python-Chemical-Engineers) - CAChemE
+    * [MO Book](https://github.com/mobook/MO-book) - Mathematical Opimization
```

Comment:
most of these don't appear to be university courses, can you move them to the appropriate sections (e.g. software, textbooks, etc.)?